apps/frontend/src/renderer/components/project-wizard/ProjectStep.tsx

```
@@ -0,0 +1,130 @@
+        {/* Options */}
+        <div className="grid grid-cols-1 md:grid-cols-2 gap-4 mb-10">
+          {/* Open Existing */}
+          <button
+            onClick={handleOpenExisting}
+            className={cn(
+              'flex items-center gap-4 p-6 rounded-xl border-2 border-border',
+              'bg-card hover:bg-accent hover:border-primary transition-all duration-200',
+              'text-left group'
+            )}
+            aria-label={t('project.openExistingAriaLabel')}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)